packages/macros/src/babel/evaluate-json.ts

```diff
@@ -347,22 +347,16 @@ export class Evaluator {
   // confidence without actually forcing the value.
   private maybeEvaluateRuntimeConfig(path: NodePath<CallExpression>): EvaluateResult {
     let callee = path.get('callee');
-    if (callee.isMemberExpression()) {
-      let prop = assertNotArray(callee.get('property'));
-      if (prop.isIdentifier() && prop.node.name === '_runtimeGet') {
```

Comment:
This `_runtimeGet` thing has been removed in #444, but in seems it has been forgotten here. 

As shown in #559, the runtime version which is now something like `config('path/to/package')` was seen by this but not recognized as such.